doc/develop/preprocessor_function.rst

```diff
@@ -244,12 +244,13 @@ and add the following content:
     """Test function `esmvalcore.preprocessor.example_preprocessor_function`."""
     from pathlib import Path
 
-    import esmvaltool_sample_data
     import iris
     import pytest
 
     from esmvalcore.preprocessor import example_preprocessor_function
 
+    esmvaltool_sample_data = pytest.importorskip("esmvaltool_sample_data")
```

Comment:
To be consistent with the actual code, e.g., https://github.com/ESMValGroup/ESMValCore/blob/fix_load_grib_test/tests/sample_data/multimodel_statistics/test_multimodel.py#L18